src/pages/TaskList/TaskList.js

```diff
@@ -38,14 +39,12 @@ export class TaskList extends React.Component {
 
     handleDoneTask = (task) => {
         task.done = true;
-        updateTask(task)
-          .then(() => this.updateTaskList());
+        this.changeTask(task);
```

Comment:
No, it is wrong again. You have two functions instead of one.
Why wouldn't make like this:
```
changeTaskState = (task, state = false) => {
        task.done = state;
        this.changeTask(task);
    };

<React.Fragment>
                                <span className="taskList__status in-progress"
                                      onClick={() => this.changeTask(task)}>~</span>
                                <span className="taskList__status delete"
                                      onClick={() => this.handleDeleteTask(task.id)}>X</span>
                                <span className="taskList__status done"
                                      onClick={() => this.changeTask(task, true)}>V</span>
                            </React.Fragment>
```